src/TestLogger/TestLogger.csproj

```diff
@@ -9,13 +9,29 @@
     <TargetFramework>netstandard1.5</TargetFramework>
     <AssemblyName>Spekt.TestLogger</AssemblyName>
     <Version>$(PackageVersion)</Version>
-    <IsPackable>false</IsPackable>
+    <IsPackable>true</IsPackable>
   </PropertyGroup>
 
   <PropertyGroup Label="Configuration">
     <RootNamespace>Spekt.TestLogger</RootNamespace>
   </PropertyGroup>
 
+  <!-- Nuget properties. Testlogger will be a local package for internal spekt consumption. -->
+  <PropertyGroup>
+    <Authors>Spekt contributors</Authors>
+    <Description>Core test logger for Spekt</Description>
+    <Copyright>Copyright 2020 © Spekt Contributors. All rights reserved.</Copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <RepositoryUrl>https://github.com/spekt/testlogger</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+<!--    <PackageIconUrl>https://raw.githubusercontent.com/spekt/testlogger/master/assets/icon-120x120.png</PackageIconUrl>-->
```

Comment:
Want me to sort out the icon for this the same as the junit one? I can do the other loggers too.

Reply:
If we decide to keep this as an internal only package icon will not not add much value. It will be awesome to add icons to other loggers :+1: